exept/exeptions.py

```diff
@@ -83,4 +83,28 @@ class UserAlreadyExistError(CustomQSportException):
     def __init__(self, date_value):
         self.date_value = date_value
         message = f"User with such data already exist'"
+        super().__init__(message)
+
+
+class NoUsersInDBError(CustomQSportException):
+    status_code = 404
+
+    def __init__(self):
+        message = f"There are no users in database"
+        super().__init__(message)
+
+
+class EmptyRecommendationListForUserError(CustomQSportException):
```

Comment:
Line 89: NoUser....


Line 97: EmptyRecc...

Keep following the same naming pattern EVERYWHERE. "NoSomethingFound" is preferred of course

Reply:
"NoRecommendationsFound" sounds better